static/app/views/dashboards/utils.tsx

```diff
@@ -586,8 +586,8 @@ export function connectDashboardCharts(groupName: string) {
   connect?.(groupName);
 }
 
-export function hasDatasetSelector(organization: Organization): boolean {
-  return organization.features.includes('performance-discover-dataset-selector');
+export function hasDatasetSelector(_organization: Organization): boolean {
+  return true;
```

Comment:
Maybe we should just delete all references to this function as well?

Reply:
yeah, i'll do that as a follow up, just wanted to keep reviews manageable - this function has a ton of references